package.json

```diff
@@ -35,7 +35,7 @@
     "@babel/preset-env": "^7.16.11",
     "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#1.3.0",
     "@duckduckgo/content-scope-utils": "github:duckduckgo/content-scope-utils#1.0.1",
-    "@playwright/test": "^1.24.1",
+    "@playwright/test": "^1.31.2",
```

Comment:
Had to bump playwright to use [`expect.toBeInViewport()`](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-in-viewport).